sql/moz-fx-data-shared-prod/fenix_derived/event_types_v1/metadata.yaml

```diff
@@ -14,7 +14,3 @@ labels:
 scheduling:
   dag_name: bqetl_fenix_event_rollup
   date_partition_parameter: null
-  parameters: ["submission_date:DATE:{{ds}}"]
```

Comment:
This manual parameter is still necessary because of `date_partition_parameter: null` and `@submission_date` being used in the ETL's query, so I'd recommend putting it back.

I manually tested generating the `bqetl_fenix_event_rollup` DAG from this branch, and this change does result in no `submission_date` parameter being passed to the ETL, but disconcertingly that change to the DAG doesn't show up in the `sql.diff` generated by CI, and I'm not sure why.

Reply:
I figured out why the diff didn't show a DAG change:  this file is overwritten by the `events_daily` SQL generator, where the associated template [does specify the parameter](https://github.com/mozilla/bigquery-etl/blob/a6297bd069fb6cac08694b926097c78fd1d038c5/sql_generators/events_daily/templates/event_types_v1/metadata.yaml#L15).

It looks like the output of the `events_daily` SQL generator is currently checked into `main`. IMO those files should either be removed to avoid confusion, or added to a `skip_existing` config if there are meaningful differences from what the SQL generator produces (probably in another PR).

Reply:
Agreed. I'll check if this is also the case for other files